client/my-sites/plans/main.jsx

```diff
@@ -36,7 +36,7 @@ const Plans = React.createClass( {
 	getDefaultProps() {
 		return {
 			intervalType: 'yearly',
-			displayJetpackPlans: true
+			displayJetpackPlans: false
```

Comment:
It was originally @gwwar's idea (https://github.com/Automattic/wp-calypso/pull/11255#discussion_r100168646) and I agreed with it because I think that there are only two types of plans which `PlansFeaturesMain` can show: either Jetpack or wpcom. If `displayJetpackPlans` is omitted, it will default to `false` (also specified in `PlansFeaturesMain` props). If somebody wants to do another Jetpack plans page with `PlansFeaturesMain`, they need to specify it to `true` and will quickly notice that it shows wpcom plans if they don't (that's why I think it's safe).